src/OpenTelemetry.Instrumentation.AspNetCore/CHANGELOG.md

```diff
@@ -35,6 +35,11 @@ exception. The attribute value will be set to full name of exception type.
 * Fixed `network.protocol.version` attribute values to match the specification.
   ([#5007](https://github.com/open-telemetry/opentelemetry-dotnet/pull/5007))
 
+* Removed filtering of metrics when the request path contained `/metrics`. This
+  change may affect prometheus pull scenario if the prometheus server sends
```

Comment:
Probably nit picking, but the wording "may affect" may not be clear as to what is the effect? The effect of this PR is - metrics now include measurements from prometheus scrape requests. So one possible effect is - users see more count and their avge, etc might be affected. Eg: the service's latency is one-digit-ms only, but prometheus scrapes take seconds.. Now, by including prometheus scrapes as to the latency metric, it may give false impression that the service slowed! It could be immaterial, if prom scrapes are a minute fraction (quite likely.)

Not a blocker for the PR!

Reply:
I reworded first line a bit more. The point about latency is valid but I don't think it falls under the instrumentation library's scope (library simply reports metrics). I think it was a bug that we were filtering measurements and not giving the full picture.